packages/components/addon/components/hds/form/field/index.js

```diff
@@ -3,7 +3,18 @@ import Component from '@glimmer/component';
 import { guid } from '../utils/guid';
 import { describedBy } from '../utils/describedby';
 
-export default class HdsFormRadioFieldComponent extends Component {
+export default class HdsFormFieldIndexComponent extends Component {
+  /**
+   * Sets the layout of the group
+   *
+   * @param layout
+   * @type {enum}
+   * @default 'vertical'
+   */
+  get fieldLayout() {
+    return this.args.fieldLayout ?? 'vertical';
```

Comment:
personal preference, but in this case I would not have a default (it's more explicit in the code what layout we're using, and in any case it's only in 3+3 places where we have to add it)

Reply:
Declared 'vertical' or 'flag' explicitly – no defaults now.